Packs/GSuiteAdmin/Integrations/GSuiteAdmin/README.md

```diff
@@ -1870,4 +1870,619 @@ There is no context output for this command.
+                    "fieldConstraints": {}
+                }
+            ],
+            "notices": [
+                {
+                    "field": "arcEnabled",
+                    "noticeValue": "true",
+                    "noticeMessage": "Before you set this to true you must accept the Managed Google Play Agreement by going to https://play.google.com/work/termsofservice with a super administrator account."
+                },
+            ],
+            "supportUri": "https://support.google.com/chrome/a/answer/9039146?hl=en#androidapps",
+            "schemaName": "chrome.users.appsconfig.AndroidAppsEnabled",
+            "validTargetResources": [
+                "ORG_UNIT"
+            ],
+            "policyApiLifecycle": {
+                "policyApiLifecycleStage": "API_CURRENT"
+            },
+            "categoryTitle": "Additional app settings",
+            "supportedPlatforms": [
+                "CHROME_OS"
+            ]
+        }
+    ]
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Policy Schemas List
+>|Name|Policy Description|Schema Name|
+>|---|---|---|
+>| customers/test/policySchemas/chrome.devices.test.appsconfig.AutoLaunchApp | Allows setting of the auto-launch app. | chrome.devices.test.appsconfig.AutoLaunchApp |
+>| customers/test/policySchemas/chrome.users.appsconfig.AndroidAppsEnabled | Android apps on Chrome devices. | chrome.devices.test.appsconfig.AutoLaunchApp |
+
+
+### gsuite-policy-resolve
+
+***
+Gets the resolved policy values for a list of policies that match a search query.
+
+#### Base Command
+
+`gsuite-policy-resolve`
+
+##### Required Permissions
+`https://www.googleapis.com/auth/chrome.management.policy`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| customer_id | ID of the Google Workspace account or literal "my_customer" for the customer associated with the request. | Required | 
+| policy_schema_filter | Specify a schema name to view a particular schema, for example: chrome.users.ShowLogoutButton, Please read https://developers.google.com/chrome/policy/guides/policy-schemas for details on schema namespaces. | Required | 
+| target_type | Type of target, can be Group or Organizational Unit. Possible values are: Group, OrgUnit. | Required | 
+| target_resource | The target resource on which this policy is applied. The following resources are supported: Organizational Unit ("orgunits/{orgunit_id}") or Group ("groups/{groupId}"). | Required | 
+| additional_target_keys | Map (key: string, value: string) containing the additional target key name and value pairs used to further identify the target of the policy. | Optional | 
+| page_size | The maximum number of policy schemas to return, defaults to 100 and has a maximum of 1000. Default is 100. | Optional | 
+| page_token | Page Token. | Optional | 
+| limit | Limit. | Optional | 
```

Comment:
Limit of what? The number of devices per retrieval? The number of devices per page?